docs/rules/no-container.md

```diff
@@ -0,0 +1,36 @@
+# Disallow the use of `container` methods (no-container)
+
+By using `container` methods like `.querySelector` you may lose a lot of the confidence that the user can really interact with your UI. Also, the test becomes harder to read, and it will break more frequently.
```

Comment:
Could you specify this applies to Testing Library frameworks built on top of DOM Testing Library please?